docs/raw/serving-community-history.md

```diff
@@ -0,0 +1,315 @@
+| Store node           | A Waku node that implements [11/WAKU-MAILSERVER](/spec/11) or [13/WAKU2-STORE](https://rfc.vac.dev/spec/13/) respectively |
+| Waku network         | A group of Waku nodes connected through the internet connection and forming a graph |
+| Community owner      | A Status user that owns a Status community |
+| Community member     | A Status user that is part of a Status community |
+| Community owner node | A Status node with message archive capabilities enabled, run by a community owner |
+| Community member node| A Status node with message archive capabilities enabled, run by a community member |
+| Live messages        | Waku messages received through the Waku network |
+
+
+## Supported platforms
+
+Creating, distributing, downloading and unpacking community history archives SHOULD only be supported in desktop clients. Mobile clients SHOULD NOT implement this functionality due to bandwidth and storage limitations.
+
+## Requirements / Assumptions
+
+This specification has the following assumptions:
+
+- Store nodes are available 24/7, ensuring constant live message availability
+- Store nodes have enough storage to persist historical messages for up to 30 days
+- No store nodes have storage to persist historical messages older than 30 days
+- All nodes are honest
+- The network is reliable
+
+Furthermore, it assumes that:
+
+- Community owner nodes have enough storage to persist historical messages older than 30 days
+- Community owner nodes provide archives with historical messages **at least** every 30 days
+- Community owner nodes receive all community messages
+- Community owner nodes are honest
+
+## Overview
+
+The following is a high-level overview of the user flow and features this specification describes.
+
+### Serving community history archives
+
+Community owner nodes go through the following (high level) process to provide community members with message histories (assumes community owner node is available 24/7):
+
+1. Community owner creates a Status community
+2. Community owner enables community history archive support
+3. A special type of channel for distributing magnet links ([Magnet URI scheme](https://en.wikipedia.org/wiki/Magnet_URI_scheme), [Extensions for Peers to Send Metadata Files](https://www.bittorrent.org/beps/bep_0009.html)) is created
+4. Community owner invites members and creates additional channels
+5. Community owner node receives messages and stores them into local database
+6. After 7 days, the community owner node exports and compresses last 7 days worth of messages from database and creates a magnet link from that data via torrent client
+7. Community owner node creates message archive index and bundles the previously generated magnet link into it
+8. Community owner node creates magnet link from index and distributes it to community members via special channel created in step 2) through the Waku network
```

Comment:
How about replacing "from index" with "from binary blob" to make it clearer what we are talking about?

Reply:
This should say "from archive index". Sorry there's a missing word...

Apart from that, as mentioned here https://github.com/status-im/specs/pull/162/files#r767957001, this would be incorrect.

We are creating a magnet link from the archive index. 

**Although** as mentioned here https://github.com/status-im/specs/pull/162/files#r767717175, creating a magnet link for the index may not be necessary, as we can distributed via waku messages as is.